runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
> I maintain that it's safer to keep the code than to remove it just because you're not sure if it's necessary -- less code being touched means a smaller surface of things that can break.

I don't understand what you want here. Are you suggesting we leave those 4 getter functions in there? We can do that only if we remove overrides on them. But if you remove overrides that will be dead code used by no one. Or are you suggesting to move the code in GetMediaRequestContextForStoragePartition() to CreateMediaRequestContextForStoragePartition()?